Documentation/SystemRequirements/PHP.rst.txt

```diff
@@ -47,6 +46,111 @@ Depending on the use case, the following extensions may also be required:
 * **openssl** (OpenSSL is required for sending SMTP mails over an encrypted channel endpoint)
 * **intl** (when using unicode-based filesystems)
 
+Opcache settings
+----------------
+
+It is recommended to enable opcache and tune the settings for optimal
+performance. Most settings will already have a good default value, but
+the settings will also depend on number of scripts, available memory etc.
+
+.. warning::
+
+   Opcache must be adapted to your specific needs. Some of the settings which
+   improve performance should not be activated for TYPO3 or come with a risk.
+   The following information has been put together carefully but it is recommended
+   to double-check the given recommendations regularly. Please help and contribute
+   to this page if you find something which is not correct!
```

Comment:
Perhaps "which can be improved" instead of "which is not correct".

Do we write incorrect documentation? ;-)

Reply:
yes, good point. Absolutely!